cookbooks/memcached/attributes/overrides.rb

```diff
@@ -0,0 +1 @@
+node.set['memcached']['version'] = '1.4.39-r1'
```

Comment:
See comment below and in https://github.com/engineyard/ey-cookbooks-stable-v5/pull/417#pullrequestreview-278305067

Reply:
Removed overrides.rb